com.ca.lsp.cobol/lsp-core-domain/src/test/java/com/broadcom/lsp/domain/cobol/event/CopybookEventsTest.java

```diff
@@ -37,6 +37,7 @@
   private static final String COBOL_FILE_URI = "file:///C:/Users/test/Main.cbl";
 
   private static final String COPYBOOK_CONTENT = "000000 IDENTIFICATION DIVISION.";
+  private static final String DID_OPEN = "DID_OPEN";
```

Comment:
you may edit the enum TextDocumentSyncType to put also a label and use it everywhere you need